commands/status.go

```diff
@@ -45,6 +47,7 @@ Agents:
 
 type statusResult struct {
 	PMMAgentStatus *agentlocal.Status `json:"pmm_agent_status"`
+	PMMVersion     string
```

Comment:
Maybe we need to add a struct field tag to be consistent.

```suggestion
	PMMVersion     string `json:"pmm_admin_version"`
```